packages/bpk-component-button/readme.md

```diff
@@ -26,7 +26,7 @@ export default () => (
     <BpkButton secondary>Secondary</BpkButton>
     <BpkButton link>Link</BpkButton>
     <BpkButton iconOnly>
-      <AlignedArrowIcon fill="white" />
```

Comment:
Just remembered that now that https://github.com/Skyscanner/backpack/pull/281 has been release, the documentation and storybook for BpkButton can be updated to remove the usage of `fill`